extensions/vscode/webviews/homeView/src/components/EvenEasierDeploy.vue

```diff
@@ -105,7 +105,14 @@
         <div class="deployment-in-progress-container">
           <div class="progress-container">
             <vscode-progress-ring class="progress-ring" />
-            Deployment in Progress...
+            <div class="progress-desc">
+              <div>Deployment in Progress...</div>
+              <div>
+                <a href="" role="button" @click="onViewPublishingLog"
+                  >View Log</a
```

Comment:
I'll also mention we have a mix of "Log" and "Logs". We should probably unify under singular or plural.

Reply:
I searched for "logs" and if they were referencing a target string seen by the user, I changed them to log. See commit: https://github.com/posit-dev/publisher/pull/1896/commits/5b5da72c1296e5518b6fc8c8ed522d129d68b6a2

Reply:
Fantastic. Thanks for unifying that.